RELEASE_PROCEDURE.md

```diff
@@ -24,68 +88,76 @@ will be working in an up-to-date local clone of the GitHub repository, where the
 
 ## Creating the Release Branch and Release Candidate
 
-When a new 3.Y.0 minor version of SingularityCE is issued the release process
-begins by branching, and then issuing a release candidate for broader testing.
-
-When a new 3.Y.Z patch release is issued, the branch will already be present,
-and steps 1-4 should be skipped.
+When a new major or minor version of SingularityCE is issued the release process
+begins by branching, and then issuing, a release candidate for broader testing.
 
-1. From a repository that is up-to-date with main, create a release branch
-   e.g. `git checkout upstream/main -b release-3.8`.
-1. Push the release branch to GitHub via `git push upstream release-3.8`.
+1. From a repository that is up-to-date with the `main` branch, create a release
+   branch e.g. `git checkout upstream/main -b release-3.12`.
+1. Push the release branch to the GitHub repository via `git push upstream
+   release-3.12`.
 1. Examine the GitHub branch protection rules, to extend them to the new release
    branch if needed.
 1. Update the `.github/dependabot.yml` configuration so that dependabot is
    tracking the new stable release branch. Do not remove the previous stable
    release branch from the configuration yet, as it should be monitored until
-   the final release of a new 3.Y.0 version.
+   the final release of a new version.
+1. Check the version of Go installed in the CI environment from
+   `.circleci/config.yml`. Ensure that it is the latest release, as it will be
+   used to build our release packages. Update if necessary via a PR against the
+   release branch.
 1. Modify the `README.md`, `INSTALL.md`, `CHANGELOG.md` via PR against the
-   release branch, so that they reflect the version to be released.
+   release branch, so that they reflect the release candidate version to be
+   issued.
 1. Apply an annotated tag via
-   `git tag -a -m "SingularityCE v3.8.0 Release Candidate 1" v3.8.0-rc.1`.
-1. Push the tag via `git push upstream v3.8.0-rc.1`.
-1. Create a tarball via `mconfig -v && make dist`.
-1. Test intallation from the tarball.
-1. Compute the sha256sum of the tarball e.g. `sha256sum *.tar.gz > sha256sums`.
-1. Create a GitHub release, marked as a 'pre-release', incorporating
-   `CHANGELOG.md` information, and attaching the tarball and `sha256sums`.
-1. Notify the community about the RC via the Google Group and Slack.
+   `git tag -a -m "SingularityCE v3.11.0 Release Candidate 1" v3.11.0-rc.1`.
```

Comment:
Same here, worth signing?

```suggestion
   `git tag -a -s -m "SingularityCE v3.11.0 Release Candidate 1" v3.11.0-rc.1`.
```

Reply:
Holding on this pending possible strategy for cross-siging / publishing keys.